backend/app/services/wework_transcript_service.py

```
@@ -0,0 +1,446 @@
+    after: int,
+    limit: int,
+) -> tuple[WeworkTranscriptArchive, list[dict[str, Any]], bool]:
+    archive = get_archive(
+        db,
+        user_id=user_id,
+        transcript_id=transcript_id,
+        archive_id=archive_id,
+    )
+    content = wework_transcript_storage.get(archive.storage_key)
+    if hashlib.sha256(content).hexdigest() != archive.sha256:
+        raise WeworkTranscriptStorageError(
```

Ruff (0.16.3)

[error] Undefined name `WeworkTranscriptStorageError`

(F821)